webpack/assets/javascripts/foreman_tools.js

```diff
@@ -10,6 +10,59 @@ import { sprintf, translate as __ } from './react_app/common/I18n';
 
 import { showLoading, hideLoading } from './foreman_navigation';
 
+import store from './react_app/redux';
+import { openConfirmModal as coreOpenConfirmModal } from './react_app/components/ConfirmModal';
+import { noop } from './react_app/common/helpers';
+
+export function openConfirmModal({
+  title = '',
+  message = '',
+  onConfirm = noop,
+  onCancel = noop,
+  modalProps = {},
+  isWarning = false,
+  confirmButtonText = null,
```

Comment:
The default values are also defined in `coreOpenConfirmModal` should they be also here?

Reply:
sure, I will add some comments above the helper for better IntelliSense